### ISSUE TITLE
2.6.0-RC1 formatting issue

### ISSUE THREAD

**Issue by anna-zzz** [olafurpg/scalafmt#310]
Original (2.5.0):
```scala
    CreditCard(customerId = customerId,
               gatewayCustomerId = sCust.getId,
               gatewayCardId = card.getId,
               holderName = p.holderName,
               lastFour = p.lastFour,
               expMonth = p.expMonth,
               expYear = p.expYear,
               isDefault = p.isDefault,
               address1Check = card.getAddressLine1Check.some,
               zipCheck = card.getAddressZipCheck.some,
               regionId = a.regionId,
               addressName = a.name,
               address1 = a.address1,
               address2 = a.address2,
               city = a.city,
               zip = a.zip,
               brand = card.getBrand)
```

Scalafmt 2.6.0-RC1:
```scala
    CreditCard(
        customerId =
          customerId,
        gatewayCustomerId =
          sCust.getId,
        gatewayCardId = card.getId,
        holderName = p.holderName,
        lastFour = p.lastFour,
        expMonth =
          p.expMonth,
        expYear = p.expYear,
        isDefault = p.isDefault,
               address1Check = card.getAddressLine1Check.some,
               zipCheck = card.getAddressZipCheck.some,
               regionId = a.regionId,
               addressName = a.name,
               address1 = a.address1,
               address2 = a.address2,
               city = a.city,
               zip = a.zip,
               brand = card.getBrand)
```

Using:
```
$ cat .scalafmt                                                                                                                                    *[bump-scalafmt] 
--style defaultWithAlign
--maxColumn 100
# --spaceAfterTripleEquals true
```

I know it's not a release bug, but just so you know stuff's broken.

**Comment by olafurpg** [comment]
Huh, I didn't expect it to be this broken. Thanks for reporting, will definitely fix before 0.2.6 release.

**Comment by olafurpg** [comment]
This issue is caused by a WIP to fix #278. For some reason the state space is exploding even though this wasn't a problem in 0.2.5.

<img width="627" alt="screen shot 2016-06-04 at 12 14 48" src="https://cloud.githubusercontent.com/assets/1408093/15798996/f1e2ceb6-2a4d-11e6-9854-a5487155528b.png">
